modules/ccalib/include/opencv2/ccalib/randpattern.hpp

```diff
@@ -42,7 +42,7 @@
 #ifndef __OPENCV_RANDOMPATTERN_HPP__
 #define __OPENCV_RANDOMPATTERN_HPP__
 
-#include "opencv2/features2d.hpp"
+#include "opencv2/features.hpp"
 #include "opencv2/xfeatures2d.hpp"
```

Comment:
@vpisarev should we rename xfeatures2d too?

Reply:
Asked @vpisarev just now, he prefers to leave`xfeatures2d` as is.

Reply:
@asmorkalov Can you change `xfeatures` in CI config for contrib back to `xfeatures2d`? `xfeatures2d` isn't renamed. Otherwise CI is all red.